ec_grasp_planner/src/RBOHandRecipesWAM.py

```diff
@@ -279,19 +281,27 @@ def create_wall_grasp(chosen_object, wall_frame, handarm_params, pregrasp_transf
     lift_hand_joint_velocity = params['max_joint_velocity']
 
     # ---- SlideToWall
-    wall_force = params['wall_force']
+    wall_force = 17 # params['wall_force']
```

Comment:
remove line

Reply:
remove or set back to wall_force = params['wall_force'] ? (I set it back for now)